account_move_exception/views/account_view.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" ?>
+<odoo>
+    <record id="action_account_test_tree" model="ir.actions.act_window">
+        <field name="name">Account Exception Rules</field>
+        <field name="res_model">exception.rule</field>
+        <field name="view_mode">tree,form</field>
+        <field name="view_id" ref="base_exception.view_exception_rule_tree" />
+        <field
+            name="domain"
+        >[('model', 'in', ['account.move', 'account.move.line'])]</field>
+        <field
+            name="context"
+        >{'active_test': False, 'default_model' : 'account.move'}</field>
+    </record>
+    <menuitem
+        id="menu_account_exception_parent"
+        parent="account.product_product_menu_sellable"
+        sequence="20"
+        name="Exceptions"
+    />
+    <menuitem
+        id="menu_account_exception_parent2"
+        parent="account.product_product_menu_purchasable"
+        sequence="20"
+        name="Exceptions"
+    />
```

Comment:
How about moving these menus to Configuration?

Reply:
Done